vapi/rest/client.go

```diff
@@ -129,6 +129,15 @@ func (c *Client) LoginByToken(ctx context.Context) error {
 	return c.Login(ctx, nil)
 }
 
+// Get returns an error if the current session is invalid.
+func (c *Client) Get(ctx context.Context) error {
+	req := internal.URL(c, internal.SessionPath).Request(http.MethodPost)
```

Comment:
You can add the query here using: `WithAction("get"`)`
Example: https://github.com/vmware/govmomi/blob/ee35ba9e79122df9873f2dc948bcce912a9c9d7f/vapi/library/library_file.go#L50